frontend/src/concepts/pipelines/content/pipelineSelector/PipelineSelectorTableRow.tsx

```diff
@@ -3,39 +3,66 @@ import { Stack, StackItem, Tooltip } from '@patternfly/react-core';
 import { Td, Tr } from '@patternfly/react-table';
 import { ExperimentKFv2, PipelineKFv2, PipelineVersionKFv2 } from '~/concepts/pipelines/kfTypes';
 import { relativeTime } from '~/utilities/time';
+import { isArgoWorkflow } from '~/concepts/pipelines/content/tables/utils';
 
 type PipelineSelectorTableRowProps = {
   obj: PipelineKFv2 | PipelineVersionKFv2 | ExperimentKFv2;
+  isSchedule?: boolean;
+  isCreatePage?: boolean;
   onClick: () => void;
 };
 
-const PipelineSelectorTableRow: React.FC<PipelineSelectorTableRowProps> = ({ obj, onClick }) => {
```

Comment:
This component deals with experiments, pipelines, and versions... however you are adding version specific logic to this component. All that logic should be moved up a level to PipelineVersionSelector. Then you can pass the row a isDisabled field and additionally a isDisabledReason.